docs/source/install/ansible.rst

```diff
@@ -88,6 +91,14 @@ Below is more advanced example to customize |st2| deployment:
         - name: Install st2web
           role: st2web
 
+        - name: Install st2chatops with "slack" hubot adapter
+          role: st2chatops
+          vars:
+            st2chatops_version: latest
+            st2chatops_st2_api_key: CHANGE-ME-PLEASE # This can be generated using "st2 apikey create -k"
+            st2chatops_hubot_adapter: slack
+            st2chatops_config: {"HUBOT_SLACK_TOKEN":"xoxb-CHANGE-ME-PLEASE"}
```

Comment:
@armab: ^^ This will not work:

This is not json, this is the hash used here: https://github.com/StackStorm/ansible-st2/blob/master/roles/st2chatops/defaults/main.yml#L17. Ansible considers it as a key value pair instead of hash:
```yaml
HUBOT_SLACK_TOKEN: xoxb-CHANGE-ME-PLEASE
```